utilities/src/main/java/io/syndesis/qe/resource/impl/Syndesis.java

```diff
@@ -444,6 +480,41 @@ public void deployOperator() {
 
         dc.getSpec().getTemplate().getSpec().getContainers().get(0).getEnv().addAll(envVarsToAdd);
 
+        List<HasMetadata> finalResourceList = resourceList;
+        OpenShiftUtils.asRegularUser(() -> OpenShiftUtils.getInstance().resourceList(finalResourceList).createOrReplace());
+
+        Map<String, String> imagesEnvVars = null;
+        // For upgrade, we want to override images only for "current" version
+        if (operatorImage.equals(TestConfiguration.syndesisOperatorImage())) {
+            imagesEnvVars = generateImageEnvVars();
+        }
+
+        if (imagesEnvVars != null && !imagesEnvVars.isEmpty()) {
+            log.info("Overriding images to be deployed");
+            try {
+                OpenShiftWaitUtils.waitFor(() -> OpenShiftUtils.getInstance().getDeploymentConfig(operatorResourcesName) != null);
+            } catch (TimeoutException | InterruptedException e) {
+                fail("Unable to get operator deployment config", e);
+            }
+            OpenShiftUtils.scale(operatorResourcesName, 0);
+
+            try {
+                OpenShiftWaitUtils.waitFor(OpenShiftWaitUtils.areNoPodsPresent(operatorResourcesName));
+            } catch (TimeoutException | InterruptedException e) {
+                fail("Operator pod shouldn't be present after scaling down", e);
+            }
+
+            Map<String, String> finalImagesEnvVars = imagesEnvVars;
+            TestUtils.withRetry(() -> {
+                try {
+                    OpenShiftUtils.getInstance().updateDeploymentConfigEnvVars(operatorResourcesName, finalImagesEnvVars);
+                    return true;
+                } catch (KubernetesClientException kce) {
+                    log.debug("Caught KubernetesClientException: " + kce);
+                    return false;
+                }
+            }, 3, 30000L, "Unable to update operator deployment config after 3 tries");
+        }
```

Comment:
:thinking: this part was changed in `1.11.x` so please adapt the changes for that

Reply:
I guess we could also remove this code and the feature file as the old operatorhub installation won't be used anymore, wdyt?

Reply:
makes sense